src/app/home/components/Waku.tsx

```diff
@@ -0,0 +1,112 @@
+import React from "react";
+import { Block } from "@/components/Block";
+import { Subtitle } from "@/components/Subtitle";
+import { Button } from "@/components/Button";
+import { MessageContent, useWaku } from "@/hooks";
+
+export const Waku: React.FunctionComponent<{}> = () => {
+  const { onSend, messages } = useWaku();
+  const { nick, text, onNickChange, onMessageChange, resetText } = useMessage();
+
+  const onSendClick = async () => {
+    await onSend(nick, text);
+    resetText();
+  };
+
+  const renderedMessages = React.useMemo(
+    () => messages.map(renderMessage),
+    [messages]
+  );
+
+  return (
+    <Block className="mt-10">
+      <Subtitle>
+        Waku
+      </Subtitle>
+
+      <Block className="mt-4">
+        <label
+          htmlFor="nick-input"
+          className="block mb-2 text-sm font-medium text-gray-900 dark:text-white"
+        >
+          Your nickname
+        </label>
+        <input
+          type="text"
+          id="nick-input"
+          placeholder="Choose a nickname"
+          value={nick}
+          onChange={onNickChange}
+          className="w-full mr-2 bg-gray-50 border border-gray-300 text-gray-900 text-sm rounded-lg focus:ring-blue-500 focus:border-blue-500 block p-2.5 dark:bg-gray-700 dark:border-gray-600 dark:placeholder-gray-400 dark:text-white dark:focus:ring-blue-500 dark:focus:border-blue-500"
+        />
+      </Block>
+
```

Comment:
Can we add some info of the nwaku node that is running under the hood?
* node info: `curl localhost:8645/debug/v1/info`
* node version: `curl localhost:8645/debug/v1/version`
* node health: `curl localhost:8645/health`

useful to know if the underlying  node is working fine.

Reply:
it's easy to add, but it I think becomes cumbersome 
maybe printing in console?

Reply:
why cumbersome? its very basic (and relevant) node information that fits in one line.

Reply:
cumbersome for interface I mean, we can hide it similarly to https://examples.waku.org/light-chat/ if you prefer